src/components/App.tsx

```diff
@@ -13,32 +13,56 @@ import PathwaysList from './PathwaysList';
 import { PathwayProvider } from './PathwayProvider';
 import { EvaluatedPathway } from 'pathways-model';
 import useGetPathwaysService from './PathwaysService/PathwaysService';
+import FHIR from 'fhirclient';
+import demoRecords from '../fixtures/MaureenMcodeDemoPatientRecords.json';
```

Comment:
```suggestion
import demoRecords from 'fixtures/MaureenMcodeDemoPatientRecords.json';
```

Use [absolute import](https://github.com/mcode/pathways/wiki/Code-Style-Guide#module-imports) unless the file is in the current or a child directory.

Reply:
Made this change.